### PR TITLE
Translations downloading from lokalise for cembra

### PR DESCRIPTION
**Pull request by mounikakonduru117**

### CODE REVIEW

File: src/translations.js [hunk 1]
```diff
@@ -92,20 +94,75 @@ module.exports = class Translation {
...
     let self = this;
     let alpha = moment();
-
-    console.info(`Spawning Translations started for ${self.config._LOCALES}`);
-
-    Promise.map(self.config._LOCALES, (translationLocale) => {
-      return self.downloadTranslationData(translationLocale).then( (transations) => {
-        self.updateCache(translationLocale, transations);
-        console.info(`Spawning Translations finished for ${translationLocale}`);
+    if(process.env.ORGANISATION=="CH"){
```
Comment (anandsahil) [inline src/translations.js]: CH is country , organisation should be cembra, you have to check what parameter's dev ops people are setting in the env

File: src/translations.js [hunk 2]
```diff
@@ -92,20 +94,75 @@ module.exports = class Translation {
...
+                  if (err) {
+                    return err;
+                  }
+                  var zip = new AdmZip(body);
+                  var zipEntries = zip.getEntries();
+                  async.each(zipEntries, function (zipContent, callback) {
+                      if(!zipContent.isDirectory){
+                        if(zipContent.name!=="en.json"){
+                          var locale=zipContent.name.split('_')[0];
+                          if(locale=="de"){
```
Comment (anandsahil) [inline src/translations.js]: this is confusing does locale de means translations are in german

Comment (mounikakonduru117) [reply src/translations.js]: Yes. For German the lokalise downloads a file with name de_CH.json   so i am replacing it with CH while storing in cache

File: src/translations.js (hunk 1 above)
Comment (mounikakonduru117) [reply src/translations.js]: 1. Ok. I will change the parameter name to country.
2. I have discussed with anil about this so we added the parameter organisation .I have informed yousuf about this. He will be informing devops people to set it in the env.

Comment (anandsahil) [reply src/translations.js]: No what i meant is that this library should be agnostic of the organisation or country, this just should loach CH config like other countries does. That is it. This is a common modules. The consumer checks which country it has to fetch.

File: src/translations.js (hunk 2 above)
Comment (anandsahil) [reply src/translations.js]: makes sense

File: src/translations.js (hunk 1 above)
Comment (mounikakonduru117) [reply src/translations.js]: So how can it be done?

File: src/translations.js [hunk 3]
```diff
@@ -94,75 +94,74 @@ module.exports = class Translation {
...
-            "original_filenames": false
-        },
-        gzip: true
-      }
-      request.post(options, function (err, data) {
-          if (err) {
-              return err;
+      console.info(`Spawning Translations started for ${self.config._LOCALES}`);
+      Promise.map(self.config._LOCALES, (translationLocale) => {
+        if(translationLocale.toLowerCase()=="ch"){
```
Comment (anandsahil) [inline src/translations.js:155]: is would be nice to have this separated out in a function, just for better code readability. Other wise implementation is ok.

**Review by anandsahil — APPROVED** [review]

**Pull request merged**